testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/actions/AppInitiatedActionUpdateProfileWithUserProfileTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.keycloak.testsuite.actions;
+
+import org.keycloak.representations.idm.RealmRepresentation;
+import org.keycloak.testsuite.forms.AbstractUserProfileTest;
+
+/**
+ * Only covers basic use cases for App Initialized actions. Complete dynamic user profile behavior is tested in {@link RequiredActionUpdateProfileWithUserProfileTest} as it shares same code as the App initialized action.   
+ * 
+ * @author Vlastimil Elias <velias@redhat.com>
+ *
+ */
+public class AppInitiatedActionUpdateProfileWithUserProfileTest extends AppInitiatedActionUpdateProfileTest {
+    
+    public AppInitiatedActionUpdateProfileWithUserProfileTest() {
+        super();
+        dynamicFormTest = true;
```

Comment:
Easier if you just override a method. Should also help once we refactor testsuite to remove test duplicates once declarative becomes the default.

Reply:
It means to override two methods, each about 20 lines of code, only to change two asserted error message. 
During refactor when declarative becomes default we have to merge all tests from this and parent, for me it seems a bit easier to remove this field and related ifs, than trying to decipher which method should stay ;-) But it's not too important to me, I can change it if you really want.

Reply:
I think I was not clear. I meant create a ìsDynamicForm` method and override it. Should be easier to track once we remove the method.